README.md

```diff
@@ -131,6 +131,10 @@ Isolator.adapters.http.disable!
 Isolator.adapters.http.enable!
 ```
 
+### Fix Offenses
+
+For the actions that should be executed only after successful transaction commit (which is mostly always so), we recommend to wrap them with the `after_commit` callback from [after_commit_everywhere] gem (or use this callback in models if it's applicable).
```

Comment:
Let's replace "recommend" with "you can try to use `after_commit` callback"; it should be considered as a quick fix, I think

Reply:
Done